tests/pages/Main.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { lightTheme, ThemeProvider } from "@deskpro/deskpro-ui";
+import { cleanup, render, waitFor } from "@testing-library/react/";
+import React from "react";
+import { Main } from "../../src/pages/Main";
+
+const renderPage = () => {
+  return render(
+    <ThemeProvider theme={lightTheme}>
+      <Main />
+    </ThemeProvider>
+  );
+};
+
+jest.mock("../../src/api/api", () => {
+  return {
+    getIncidentsById: () => [
+      {
+        incident: {
+          id: 1,
+          priority: {
+            summary: "P0",
+            id: "0",
+          },
+          urgency: "high",
+          status: "resolved",
+          assignments: [
+            {
+              assignee: {
+                summary: "John Doe",
+                id: "1",
+              },
+            },
+          ],
+          incident_key: "123456789",
+          service: {
+            summary: "Default Service",
+          },
+          description: "This is a description",
+          created_at: "2021-01-01T00:00:00Z",
+          resolved_at: "2021-01-01T00:00:00Z",
+          title: "This is a title",
+        },
+      },
+    ],
+  };
+});
+
+describe("Main", () => {
+  test("Main page should show all data correctly", async () => {
```

Comment:
If nothing is found, how will it work?

Reply:
If nothing is found, it will return an empty array. ![] is false

Reply:
And it will just redirect to the link page